### PR TITLE
Make server section full width

### PR DESCRIPTION
**Pull request by grahambancroft**
## Done

Make server section full width

## QA

This is just adding the additional div to make the rows full width. There will be QA issues such as misplaced images and rows needing the additional class of strip-light. A code review would probably be sufficient at this stage.

## Issue / Card

Card: https://trello.com/c/6aU6HEyP

### CODE REVIEW

File: templates/server/index.html [hunk 1]
```diff
@@ -6,352 +6,383 @@
...
-
-<div class="row row-grey equal-height">
-    <div class="eight-col equal-height__item">
-        <h2>Ubuntu Server on Power</h2>
-        <p>In addition to x86 and ARM servers, Ubuntu is supported on the Power architecture. For the enterprise datacentre, this means you can now build your infrastructure on any hardware you choose.</p>
-    </div>
-    <div class="four-col last-col equal-height__item equal-height__align-vertically align-center">
-        <img src="{{ ASSET_SERVER_URL }}683950fd-logo-ibm.svg" alt="IBM logo" width="200" />
+<section class="row row-her equal-height">
+    <div class="strip-inner-wrapper">
```
Comment (anthonydillon) [inline templates/server/index.html]: Does equal-height need to move to this div?

File: templates/server/index.html [hunk 2]
```diff
@@ -6,352 +6,383 @@
...
-    <div class="six-col equal-height__item">
-        <h2 id="the-fast-track-to-virtualisation">The fast track to virtualisation and containers</h2>
-        <p>Ready to boost efficiencies and reduce costs? Virtualise your servers with Ubuntu Server, KVM and LXD. When you use a secure, lean version of Ubuntu as a guest operating system for your application, you can create virtual machines and machine containers in seconds. KVM, LXD, Xen, VMware, Vagrant, VirtualBox, and Docker are all first class experiences with Ubuntu Server. KVM virtualisation is now also available in Ubuntu Server for ARM and IBM POWER8.</p>
+</section>
 
-        <p>Canonical supports an open ecosystem of operating system intercompatibility. By providing choice, users of Ubuntu for both servers and clouds are empowered to succeed with any workload. We encourage others to do the same and offer support for all (currently supported) versions of Ubuntu Server running on other virtualisation platforms, including KVM on other commercially available Linux distributions, VMware vSphere, and Microsoft Hyper-V.</p>
+<section class="row row-grey equal-height">
+    <div class="strip-inner-wrapper">
```
Comment (anthonydillon) [inline templates/server/index.html]: Does `equal-height` need to be on this element?

File: templates/server/index.html [hunk 3]
```diff
@@ -6,352 +6,383 @@
...
-            <li class="mailing-lists"><a href="https://lists.ubuntu.com/">Mailing lists</a></li>
-            <li class="ubuntu-blog"><a href="https://insights.ubuntu.com/">Ubuntu Insights</a></li>
-            <li class="ask-ubuntu"><a href="http://ubuntuforums.org">Ubuntu Forums</a></li>
-        </ul>
+</section>
+
+{% include "shared/_release_schedule.html" %}
+
+<section class="row row-support-systems-management equal-height">
+    <div class="strip-inner-wrapper">
```
Comment (anthonydillon) [inline templates/server/index.html]: Does `equal-height` need to be on this elememt?

File: templates/server/index.html [hunk 4]
```diff
@@ -6,352 +6,383 @@
...
+                <li class="inline-logos__item">
+                    <img class="inline-logos__image" src="{{ ASSET_SERVER_URL }}6697a21a-suse-logo.png" width="100" alt="Suse logo">
+                </li>
+            </ul>
+        </div>
+    </div>
+</section>
 
+<section class="row row-grey equal-height no-border">
+    <div class="strip-inner-wrapper">
```
Comment (anthonydillon) [inline templates/server/index.html]: Does `equal-height` need to be on this element?

**Comment by grahambancroft** [comment]
@anthonydillon I was going to hit all the equal-height instances on the next pass, although I may have done some already

**Comment by anthonydillon** [comment]
Looks good :+1:

**Pull request merged**